kolibri/core/assets/src/utils/vue-intl-locale-data.js

```diff
@@ -4,123 +4,25 @@
-          resolve(require('vue-intl/locale-data/my.js'));
-        });
-      });
-    case 'nyn':
-      return new Promise(function(resolve) {
-        require.ensure(['vue-intl/locale-data/nyn.js'], function(require) {
-          resolve(require('vue-intl/locale-data/nyn.js'));
-        });
-      });
-    case 'pt-BR':
-      return new Promise(function(resolve) {
-        require.ensure(['vue-intl/locale-data/pt.js'], function(require) {
-          resolve(require('vue-intl/locale-data/pt.js'));
-        });
-      });
-    case 'sw-TZ':
-      return new Promise(function(resolve) {
-        require.ensure(['vue-intl/locale-data/sw.js'], function(require) {
-          resolve(require('vue-intl/locale-data/sw.js'));
-        });
-      });
-    case 'ta':
-      return new Promise(function(resolve) {
-        require.ensure(['vue-intl/locale-data/ta.js'], function(require) {
-          resolve(require('vue-intl/locale-data/ta.js'));
-        });
-      });
-    case 'te':
-      return new Promise(function(resolve) {
-        require.ensure(['vue-intl/locale-data/te.js'], function(require) {
-          resolve(require('vue-intl/locale-data/te.js'));
-        });
-      });
-    case 'ur-PK':
-      return new Promise(function(resolve) {
-        require.ensure(['vue-intl/locale-data/ur.js'], function(require) {
-          resolve(require('vue-intl/locale-data/ur.js'));
-        });
-      });
-    case 'yo':
-      return new Promise(function(resolve) {
-        require.ensure(['vue-intl/locale-data/yo.js'], function(require) {
-          resolve(require('vue-intl/locale-data/yo.js'));
-        });
-      });
-    case 'zu':
-      return new Promise(function(resolve) {
-        require.ensure(['vue-intl/locale-data/zu.js'], function(require) {
-          resolve(require('vue-intl/locale-data/zu.js'));
-        });
-      });
-    default:
-      return Promise.resolve({});
-  }
+module.exports = function() {
+  const data = [];
+  data.push(require('vue-intl/locale-data/ar.js'));
+  data.push(require('vue-intl/locale-data/bn.js'));
+  data.push(require('vue-intl/locale-data/en.js'));
+  data.push(require('vue-intl/locale-data/es.js'));
+  data.push(require('vue-intl/locale-data/fa.js'));
+  data.push(require('vue-intl/locale-data/fr.js'));
+  data.push(require('vue-intl/locale-data/hi.js'));
+  data.push(require('vue-intl/locale-data/kn.js'));
+  data.push(require('vue-intl/locale-data/mr.js'));
+  data.push(require('vue-intl/locale-data/my.js'));
+  data.push(require('vue-intl/locale-data/nyn.js'));
+  data.push(require('vue-intl/locale-data/pt.js'));
+  data.push(require('vue-intl/locale-data/sw.js'));
+  data.push(require('vue-intl/locale-data/ta.js'));
+  data.push(require('vue-intl/locale-data/te.js'));
+  data.push(require('vue-intl/locale-data/ur.js'));
+  data.push(require('vue-intl/locale-data/yo.js'));
+  data.push(require('vue-intl/locale-data/zu.js'));
+  return data;
```

Comment:
This will work for now, but I worry about the sustainability of this approach in the long term.

Reply:
Yes agreed. In an upcoming PR I'll have a similar issue with font subset files for each language.

Eventually I'd like to get your help with figuring out how to inline individual files based on the current language, but for now loading everything is still faster than adding an async request.

(Note this will only become critical when we add Chinese, Japanese, or Korean because these font files are enormous)